daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/BlindingCoder.scala

```diff
@@ -25,13 +25,6 @@ object BlindingCoder {
           parties <- toPartySet(n.getPartiesList)
         } yield ni -> parties)
 
-    val implicitLocal =
-      p.getLocalImplicitDisclosureList.asScala.map(n =>
```

Comment:
This field still exists on the protobuf message. We're now just ignoring it. Is that the right thing to do or should we rather assert that it is empty?

Reply:
I'm not entirely sure. Could an assert end up causing issues with backwards compatibility?

Reply:
If it caused issues with backward compatibility, then ignoring the field would be the wrong thing, wouldn't it? @gerolf-da @remyhaemmerle-da do you know more?

Reply:
I don't think we use the `BlindingInfo` protobuf message anywhere. Whenever `com.daml.lf.transaction.BlindingInfo` is needed, it is computed from the `Transaction` itself by using one of the `com.daml.lf.engine.Blinding` functions.
@remyhaemmerle-da can you confirm that? Might be worth checking with other ledger integrations to be sure though.

Reply:
I was not really aware of this proto encoding.
It seems to be completly dead code with the `BlindingCoder` class and a test.

Reply:
I push a commit where I drop this useless file.